### PR TITLE
[release-1.17] dual-stack blog for Chinese

### PR DESCRIPTION
**Pull request by zhlsunshine**
Manually cherry pick blog for 1.17 release.
Fixes: [issue#12884](https://github.com/istio/istio.io/issues/12884)

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2023/experimental-dual-stack/index.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+title: "支持双栈 Kubernetes 集群"
+description: "对双栈 Kubernetes 集群的实验性支持。"
+publishdate: 2023-03-10
+attribution: "张怀龙 (Intel), 徐贺杰 (Intel), 丁少君 (Intel), Jacob Delgado (F5), 蔡迎春 (前 F5)"
+keywords: [双栈]
+---
+
+在过去的一年里，英特尔和 F5 在为 Istio 提供 [Kubernetes 双栈网络](https://kubernetes.io/docs/concepts/services-networking/dual-stack/)的支持中通力合作。
+
+## 背景
```
Comment (rootsongjc) [inline content/zh/blog/2023/experimental-dual-stack/index.md:11]: Please add an anchor for the header.

Comment (zhlsunshine) [reply content/zh/blog/2023/experimental-dual-stack/index.md:11]: Hi @rootsongjc, what do you mean the anchor for the header?

Comment (zhlsunshine) [reply content/zh/blog/2023/experimental-dual-stack/index.md]: Hi @rootsongjc, okay, done, thanks!

**Comment by kfaseela** [comment]
@rootsongjc okay to merge?

**Review by windsonsea — APPROVED** [review]
Looks all right except for some minors that can be improved later

File: content/zh/blog/2023/experimental-dual-stack/index.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+fd00:10:244::1a:9000     HEALTHY     OK                outbound|9000||tcp-echo.dual-stack.svc.cluster.local
+fd00:10:244::18:9000     HEALTHY     OK                outbound|9000||tcp-echo.ipv6.svc.cluster.local
+{{< /text >}}
+
+## 参与其中{#get-involved}
+
+还有很多工作要做，欢迎各位与我们一起完成双栈特性到达 Alpha 状态所需的其他任务。 [详情请看这里](https://github.com/istio/enhancements/pull/141)
+比如，来自英特尔的丁少君和李纯已经就 ambient 的网络流量重定向功能与社区一起展开工作。我们希望在后面的 Istio 1.18 alpha 双栈特性的版本中，ambient 也能够支持双栈特性。
+
+我们非常乐意你提出宝贵意见，如果你期待与我们合作请访问我们在 [Istio Slack](https://slack.istio.io/) 中的 Slack 频道 **#dual-stack-support**。
```
Comment (kebe7jun) [inline content/zh/blog/2023/experimental-dual-stack/index.md:170]:
```suggestion
我们非常乐意您提出宝贵意见，如果您期待与我们合作请访问我们在 [Istio Slack](https://slack.istio.io/) 中的 **#dual-stack-support** 频道。
```

**Comment by rootsongjc** [comment]
> @rootsongjc okay to merge?

yes, can do.

**Comment by zirain** [comment]
@zhlsunshine please sync changes to master branch.

**Pull request merged**